migrations/V1__Add_System_Intake.sql

```diff
@@ -0,0 +1,26 @@
+create table system_intake (
+    id uuid PRIMARY KEY not null,
+    eua_user_id text not null,
+    requester text,
+    component text,
+    business_owner text,
+    business_owner_component text,
+    product_manager text,
+    product_manager_component text,
+    has_isso bool,
+    isso text,
+    has_trb_collaborator bool,
```

Comment:
Looking at the ticket for validations and it reads:
```
If the user selects "1 or more of the following in OIT (select all that apply)" and does not check any of the checkboxes, display message:

    "Mark all teams you are currently collaborating with"
    If any of the checkboxes (for TRB, OIT security and/or EA are checked and no entry is enter in the subsequent text boxes, display message:
        "Tell us the name of the person you've been working with"
```
I'm wondering if this means the frontend won't send this data if those validation errors exist. This would mean, we don't need those fields, since they'll never happen independently of the name being entered, but bring up another question around autosaving work with validation errors. Curious how we want to handle that.

Reply:
@mikena-truss I think it depends on if we want to save user state the exact same as they left it, even if there will be validation errors. Do we validate before saving?
cc @christopherhuii

Reply:
I think a user should be able to save invalid data. If a user can not save invalid data, that defeats the purpose of auto saving. The user needs to be assured that their progress is able to be saved.

We should not be validating data when the user saves.